roles/openshift_node/tasks/upgrade/bootstrap_changes.yml

```diff
@@ -63,6 +43,31 @@
   - "{{ openshift.common.config_base }}/node/node.kubeconfig"
   - "{{ openshift.common.config_base }}/node/node-config.yaml"
 
+- name: Update node-config to prepare for bootstrapping
```

Comment:
Moved later in the file to guarantee we apply these to the bootstrap-node-config